frontend/ui/src/pages/Inbox/SuggestedReplySelector/index.module.scss

```diff
@@ -33,7 +33,7 @@
   width: max-content;
   height: auto;
   flex-shrink: 0;
-  max-width: 320px;
+  max-width: 400px;
```

Comment:
I thought we were removing this, what about the carousels?

Reply:
works smoothly on big screens, if you use it with 13' the experience is broken without max-width. @AitorAlgorta